egs/wsj/s5/steps/nnet3/train_discriminative.sh

```diff
@@ -307,24 +303,25 @@ while [ $x -lt $num_iters ]; do
       nnet3-am-copy --set-raw-nnet=- $dir/$x.mdl $dir/$[$x+1].mdl || exit 1;
 
     rm $nnets_list
+    [ ! -f $dir/$[$x+1].mdl ] && echo "$0: Did not create $dir/$[$x+1].mdl" && exit 1;
+    if [ -f $dir/$[$x-1].mdl ] && $cleanup && \
+       [ $[($x-1)%$keep_model_iters] -ne 0  ] && \
```

Comment:
I made the default as 100. keep_model_iters keeps models permanently. The remove models stage at the end also uses the same shift. This is same as the code that is used in train_tdnn.sh.